index.js

```diff
@@ -44,12 +44,12 @@ var pluralTypes = {
   chinese: function () { return 0; },
   german: function (n) { return n !== 1 ? 1 : 0; },
   french: function (n) { return n > 1 ? 1 : 0; },
-  russian: function (n) {
+  russian_coatian: function (n) {
```

Comment:
this seems like a typo?

Reply:
addressed the change.  I'm still waiting back from the translators but i added the test and made the changes to reflex your comments.